lib/loopback.js

```diff
@@ -76,6 +96,14 @@ function createApplication() {
   app.connector('memory', loopback.Memory);
   app.connector('remote', loopback.Remote);
 
+  if (loopback.localRegistry || options && options.localRegistry === true) {
+    // setup the app registry
+    var registry = app.registry = new Registry();
+    require('./builtin-models')(registry);
```

Comment:
It would be nice if `localRegistry` was initialized only with `Model` and `PersistedModel`. That will make it easier to exclude unused built-in models from the browser bundle. I want to make this change in 3.0 anyways, having it in 2.x under a flag would make migration from 2.x to 3.0 easier.